server/gfmd/user.h

```diff
@@ -12,14 +12,13 @@ struct user *user_lookup_in_tenant_including_invalid(
 	const char *, struct tenant *);
 struct user *user_lookup_in_tenant(const char *, struct tenant *);
 struct user *user_lookup_gsi_dn(const char *);
-struct user *user_lookup_auth_id(enum auth_user_id_type, const char *);
+struct user *user_lookup_auth_id(const char *, const char *);
```

Comment:
Please keep the type of first parameter of user_lookup_auth_id() as enum auth_user_id_type,
and export the definitione of the enum at user.h